Gemfile.lock

```diff
@@ -44,31 +44,31 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.0.0)
       rails (~> 6.0)
-    activeadmin (2.7.0)
+    activeadmin (2.8.1)
```

Comment:
### [activeadmin](https://activeadmin.info)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.7.0 → 2.8.1

[_~~change-log~~, [source-code](http://github.com/activeadmin/activeadmin)_]





<details>
<summary>Commits</summary>

A change of **301** commits. See the full changes on [the compare page](https://github.com/activeadmin/activeadmin/compare/v2.7.0...v2.8.1).

These are the first 10 commits:
- (9375b8f) [Enable `Style/FirstArgumentIndentation` cop](https://github.com/activeadmin/activeadmin/commit/9375b8ff105004016f550e88ee03c8d54eafa714)
- (5ffa6a9) [Enable `Layout/ParameterAlignment` cop](https://github.com/activeadmin/activeadmin/commit/5ffa6a92a56570685022a3af0f00b7327cfaed82)
- (4ffd744) [Enable `Layout/FirstHashElementIndentation` cop](https://github.com/activeadmin/activeadmin/commit/4ffd7442ed1b81c7b04d1e90aa25f6e1e3d62758)
- (9a99260) [Enable `Layout/HashAlignment` cop](https://github.com/activeadmin/activeadmin/commit/9a99260df4c29ac2a2c91e9f75fa9d63c13cb2f2)
- (805e84e) [Bump rubocop from 0.83.0 to 0.84.0 (#6246)](https://github.com/activeadmin/activeadmin/commit/805e84eb8415e92e51dd422dfc9fec1a372e93d0)
- (3bc0b5a) [Merge pull request #6245 from activeadmin/hashes_style](https://github.com/activeadmin/activeadmin/commit/3bc0b5a5472926de3c27821cdf8009177fe458b9)
- (2c35380) [Bump github-pages from 204 to 206 in /docs (#6251)](https://github.com/activeadmin/activeadmin/commit/2c35380954fe736bff945eefc6e20c6911b2c7e9)
- (e1d9bd2) [Bump @babel/core from 7.9.6 to 7.10.0 (#6254)](https://github.com/activeadmin/activeadmin/commit/e1d9bd2963e59d9dfbbf46fa2571d601da15b30a)
- (108c8d4) [Bump @babel/preset-env from 7.9.6 to 7.10.0 (#6253)](https://github.com/activeadmin/activeadmin/commit/108c8d42e1df553bb73988b5d190daec7bfcb0da)
- (9b18b57) [Fix a bunch of kwars warnings (#6237)](https://github.com/activeadmin/activeadmin/commit/9b18b5786866991f56e5d8cd068abcd5a861bd0c)


</details>